integration-cli/docker_api_build_test.go

```diff
@@ -16,19 +16,9 @@ import (
 
 func (s *DockerSuite) TestBuildAPIDockerFileRemote(c *check.C) {
 	testRequires(c, NotUserNamespace)
-	var testD string
-	if testEnv.DaemonPlatform() == "windows" {
-		testD = `FROM busybox
-COPY * /tmp/
-RUN find / -name ba*
-RUN find /tmp/`
-	} else {
-		// -xdev is required because sysfs can cause EPERM
-		testD = `FROM busybox
-COPY * /tmp/
-RUN find / -xdev -name ba*
-RUN find /tmp/`
-	}
+	testD := `FROM busybox
```

Comment:
Previous implementation created a fake context tar after pulling the `Dockerfile` as having a tar was the only way to start the builder(the implementation didn't bother to create `.dockerignore` as well so the `Dockerfile` would not show up). File `baz` does not exist but is mentioned in the URL. I'm not sure how it would leak, but I don't actually remember why I decided to change the `Dockerfile` itself. I'll try to change it back and put back the `baz` assertion.